### PR TITLE
[BPK-2938]: Added title only support by making subtitle optional

### PR DESCRIPTION
**Pull request by ojcurtis**
Added support to the `BpkRating` component to support title only scenario.

Screenshot 2019-09-13 at 16 02 54

### CODE REVIEW

File: UNRELEASED.yaml [hunk 1]
```diff
@@ -24,3 +24,7 @@
 #   FIXED:
 #     - bpk-component-horcrux:
 #       - Fixed issue where `BpkHorcrux` would occasionally possess the living.
+
+ADDED:
+  - bpk-component-rating:
+    - Made `subtitle` prop optional to support title only variant of rating component.
```
Comment (shaundon) [inline UNRELEASED.yaml]:
```suggestion
    - Made `subtitle` prop optional to support title-only variant of rating component.
```

File: packages/bpk-component-rating/README.md [hunk 2]
```diff
@@ -30,10 +29,10 @@ export default () => (
 | --------- | --------------------- | -------- | ----------------- |
 | ariaLabel | string                | true     | -                 |
 | title     | string                | true     | -                 |
-| subtitle  | string                | true     | -                 |
 | value     | number                | true     | -                 |
 | className | string                | false    | null              |
 | size      | oneOf(RATING_SIZES)   | false    | RATING_SIZES.base |
+| subtitle  | string                | false    | -                 |
```
Comment (shaundon) [inline packages/bpk-component-rating/README.md]:
```suggestion
| subtitle  | string                | false    | null                 |
```

Default value should be null

File: packages/bpk-component-rating/src/BpkRating.js [hunk 3]
```diff
@@ -80,6 +80,10 @@ const BpkRating = (props: Props) => {
     textStyles.push(getClassName('bpk-rating__text--horizontal'));
   }
 
+  // if(!subtitle) {
+
+  // }
+
```
Comment (shaundon) [inline packages/bpk-component-rating/src/BpkRating.js]: Should this be here?

Comment (ojcurtis) [reply packages/bpk-component-rating/src/BpkRating.js]: Ooops good spot!

File: UNRELEASED.yaml (hunk 1 above)
Comment (ojcurtis) [reply UNRELEASED.yaml]: Done!

File: packages/bpk-component-rating/src/BpkRating.js (hunk 3 above)
Comment (ojcurtis) [reply packages/bpk-component-rating/src/BpkRating.js]: Done!

File: packages/bpk-component-rating/README.md (hunk 2 above)
Comment (ojcurtis) [reply packages/bpk-component-rating/README.md]: Done!

**Pull request merged**